apps/pie-storybook/stories/pie-input.stories.ts

```diff
@@ -296,44 +299,89 @@ const Template = ({
     `;
 };
 
-const FormTemplate: TemplateFunction<InputProps> = (props: InputProps) => {
-    function onSubmit (event: SubmitEvent) {
-        event.preventDefault();
+const WithLabelTemplate: TemplateFunction<InputProps> = (props: InputProps) => html`
+        <p>Please note, the label is a separate component. See <pie-link href="/?path=/story/form-label">pie-form-field</pie-link>.</p>
+        <pie-form-label for="${props.name}">Label</pie-form-label>
+        ${Template(props)}
+    `;
 
-        const form = event.target as HTMLFormElement;
-        const formData = new FormData(form);
+const ExampleFormTemplate: TemplateFunction<InputProps> = () => html`
+    <style>
+        .form {
+            display: flex;
+            flex-direction: column;
+            padding: var(--dt-spacing-d);
+        }
 
-        const data: { [key: string]: string } = {};
-        formData.forEach((value, key) => {
-            data[key] = value.toString();
-        });
+        .form-field {
+            display: block;
+            margin-bottom: var(--dt-spacing-d);
+        }
 
-        action('submit')({
-            ...data,
-        });
-    }
+        .form-field[type="number"],
+        .form-field[type="tel"] {
+            width: 25ch;
+        }
 
-    return html`
-        <h2>Fake form</h2>
-        <form id="testForm" @submit="${onSubmit}">
-            <section>
-                <h2>Contact information</h2>
-                <p>
-                    ${props.name ? html`<pie-form-label for="${props.name}">${props.name}</pie-form-label>` : nothing}
-                    ${Template({ ...props, type: 'text' })}
-                </p>
-            </section>
-            <section style="display: flex; gap: var(--dt-spacing-a); justify-content: flex-end; flex-wrap: wrap; margin-top: var(--dt-spacing-b);">
-                <pie-button type="reset" variant="secondary">Reset</pie-button>
-                <pie-button type="submit" variant="primary">Submit</pie-button>
-            </section>
-        </form>
-    `;
-};
+        .form-btns {
+            margin-top: var(--dt-spacing-c);
+            display: flex;
+            gap: var(--dt-spacing-a)
+        }
+
+        .form-btns>.form-btn:first-of-type {
```

Comment:
Super nit, if you're gonna make any other updates you can put this one in with it
```suggestion
        .form-btns > .form-btn:first-of-type {
```

Reply:
weirdly I am almost certain some linter did the previous! Will commit yours, cheers